packages/zoe/src/contracts/simpleExchange.js

```diff
@@ -14,134 +20,135 @@ import { makeZoeHelpers, defaultAcceptanceMsg } from '../contractSupport';
  * Price is a limit that may be improved on. This simple exchange does
  * not partially fill orders.
  */
-// zcf is the Zoe Contract Facet, i.e. the contract-facing API of Zoe
-export const makeContract = harden(zcf => {
-  let sellOfferHandles = [];
-  let buyOfferHandles = [];
-  const { notifier, updater } = produceNotifier();
-
-  const {
-    rejectOffer,
-    checkIfProposal,
-    swap,
-    canTradeWith,
-    getActiveOffers,
-    assertKeywords,
-    inviteAnOffer,
-  } = makeZoeHelpers(zcf);
-
-  assertKeywords(harden(['Asset', 'Price']));
-
-  function flattenOffer(o) {
-    return {
-      want: o.proposal.want,
-      give: o.proposal.give,
-    };
-  }
-
-  function flattenOrders(offerHandles) {
-    const result = zcf
-      .getOffers(zcf.getOfferStatuses(offerHandles).active)
-      .map(offerRecord => flattenOffer(offerRecord));
-    return result;
-  }
-
-  function getBookOrders() {
-    return {
-      buys: flattenOrders(buyOfferHandles),
-      sells: flattenOrders(sellOfferHandles),
-    };
-  }
+export const makeContract = harden(
+  /** @param {ContractFacet} zcf */ zcf => {
```

Comment:
This comment and the changes above are the only non-white spaces changes in the file. The rest are driven by eslint.  This declaration allows vscode to typecheck arguments to all calls on zcf functions.